src/renderer/components/cluster-manager/cluster-manager.tsx

```diff
@@ -56,6 +57,7 @@ export class ClusterManager extends React.Component {
         <main>
           <div id="lens-views"/>
           <Switch>
+            <Redirect exact from={catalogURL()} to={`${catalogURL()}/${previousActiveTab.get() || browseCatalogPage}`}/>
```

Comment:
`catalogURL({group: previousActiveTab.get()})` + maybe add `browseCatalogPage` as default value for `previousActiveTab`?

Reply:
It is a default value, but adding extra check here for users already having empty string ("") for `previousActiveTab` in their local storage. This check overrides such case.